app/windows/Storage/Components/StorageElement.jsx

```diff
@@ -44,9 +44,11 @@ class StorageElement extends React.Component {
       {
         label: 'Publish to IPNS',
         click: (item) => {
-          publishToIPNS(this.props.element.hash)
+          const el = this.props.element
+          publishToIPNS(el.hash)
             .then(result => {
-              const message = `IPNS ${result.name} has been successfully updated to ${result.value}!`
+              const newHash = `${el.hash} - ${el.stat.CumulativeSize.value} ${el.stat.CumulativeSize.unit}`
```

Comment:
I guess that the hash is enough, but if we can get the file names it would be helping a lot to understand
what they have published! (maybe remove the size).